src/pages/MetricDetail.svelte

```diff
@@ -256,18 +256,25 @@
               content={"Explore this metric in Mozilla's instance of Looker."}
             />
           </td>
-          <td>
-            <div>
-              In
-              <AuthenticatedLink href={pingData.looker.base.url}>
-                {pingData.looker.base.name}
-              </AuthenticatedLink>
-              as
-              <AuthenticatedLink href={pingData.looker.metric.url}>
-                {pingData.looker.metric.name}
-              </AuthenticatedLink>
-            </div>
-          </td>
+          {#if metric.send_in_pings.length === 1 && metric.send_in_pings[0] === "events" && metric.type !== "event"}
```

Comment:
Note that, since metric fields sent as part of an event ping are not available in Looker, any metric type other than `event` will not be available to explore in `glean_event_counts`.